### ISSUE TITLE
Forward the email

### ISSUE THREAD

**Issue by tracktor-git** [amirsanni/php-ews-wrapper#3]
Hi there!

Is it possible to forward an incoming email to another email address? How can i do this?

**Comment by amirsanni** [comment]
No, there is no "Forward" functionality. You can, however, get the details of a message and send it to another recipient.
